src/otx/algo/action_classification/backbones/movinet.py

```diff
@@ -10,7 +11,6 @@
 import torch
 import torch.nn.functional as F  # noqa: N812
 from einops import rearrange
-from mmaction.models import MODELS
 from mmengine.config import Config
```

Comment:
mmengine dependency should be removed.

Reply:
thanks. removed at ab701960e6a342c8fcbbdb6f72e2e61cb24e0ff1